M2/Macaulay2/packages/Macaulay2Doc/functions/setupLift-doc.m2

```diff
@@ -0,0 +1,43 @@
+--- status: DRAFT
+--- author(s): PZJ
+--- notes: 
+
+undocumented methods setupLift
+
+doc ///
+ Key
+  setupLift
+ Headline
+  automatic lift from one ring to another
```

Comment:
In what sense is this "automatic"? Seems like the purpose of this function is to _manually_ setup lifting method.

Reply:
oops. copy-pasted too quickly the `setupPromote` doc.

Reply:
My complaint applies there, too. How is that automatic?

Reply:
it is automatic in the sense that you don't need to invoke `promote` explicitly, see the examples provided for `setupPromote`